config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  get '/access-requests' => 'access_requests#index'
-  get '/organisations' => 'organisations#index'
+  root to: redirect('/organisations-engagement-report')
+
+  get '/access-requests', to: 'access_requests#index'
+  get '/organisations', to: 'organisations#index'
```

Comment:
For rails convention this could be 
```
resources :access_requests, path: 'access-requests', only: :index
resources :organisations, only: :index
```

Reply:
I think I'd do that once there are a few more actions for each controller, for the moment I find the explicit declarations a bit more expressive.